internal/driver/capabilities.go

```diff
@@ -56,6 +58,8 @@ func NodeServiceCapabilities() []*csi.NodeServiceCapability {
 func VolumeCapabilityAccessModes() []*csi.VolumeCapability_AccessMode {
 	modes := []csi.VolumeCapability_AccessMode_Mode{
 		csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER,
+		csi.VolumeCapability_AccessMode_SINGLE_NODE_SINGLE_WRITER,
+		csi.VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER,
```

Comment:
Does `SINGLE_NODE_MULTI_WRITER` access mode work out of box with our current CSI driver (node server) implementation?

Reply:
`SINGLE_NODE_MULTI_WRITER` is the same as previous `SINGLE_NODE_WRITER` so yes :)
That's a trick that can be used to have multiple pods using the same PV... if there are pinned on the same node with affinity ;)